SmartDeviceLink/SDLProxy.m

```diff
@@ -174,7 +174,7 @@ - (void)sdl_sendMobileHMIState {
     }
 
     SDLLogD(@"Mobile UIApplication state changed, sending to remote system: %@", HMIStatusRPC.hmiLevel);
-    [self sendRPC:HMIStatusRPC];
+    [self sendRPC:HMIStatusRPC withEncryption:NO];
```

Comment:
Anything with `NO` here and below will fail if encryption is required. This is probably more difficult to fix in this class than in the Lifecycle Manager because this class doesn't check which RPCs require encryption and which don't. You can ping me if you want to talk about how to deal with this.